hadoop-hdds/rocksdb-checkpoint-differ/src/main/java/org/apache/ozone/rocksdiff/RocksDBCheckpointDiffer.java

```diff
@@ -163,6 +178,8 @@ public class RocksDBCheckpointDiffer implements AutoCloseable,
   private final long maxAllowedTimeInDag;
   private final BootstrapStateHandler.Lock lock
       = new BootstrapStateHandler.Lock();
+  private final Striped<ReadWriteLock> stripedSSTLock;
```

Comment:
Use IOzoneManagerLock directly. Add a resource SST_FILE_LOCK
https://github.com/apache/ozone/blob/0f52a34d0c45f5f17a568ea0d183880dfcdcd063/hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/lock/OzoneManagerLock.java#L571
https://github.com/apache/ozone/blob/6cce2bb5d2f8bd6b6f54362fcb256ffc5c1b2e2a/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java#L924-L927

Reply:
Ozone Manager jar has a dependency rocksdb-checkpoint-differ jar to be compiled first. So, I can't reference `org.apache.hadoop.ozone.om.lock.OzoneManagerLock` packaged in the ozone manager jar in `org.apache.ozone.rocksdiff.RocksDBCheckpointDiffer`